solr/contrib/jaegertracer-configurator/src/test/org/apache/solr/jaeger/TestJaegerConfigurator.java

```diff
@@ -37,8 +36,7 @@
 
 public class TestJaegerConfigurator extends SolrTestCaseJ4 {
 
-  @Rule
-  public TestRule solrTestRules = new SystemPropertiesRestoreRule();
+  @Rule public TestRule solrTestRules = new SystemPropertiesRestoreRule();
```

Comment:
Is this option configurable?

Reply:
> Is this option configurable?

Experimentally if `solrTestRules` is changed to be a different name so that the line with the `@Rule` prepended is quite a bit longer then it leaves the `@Rule` on the previous line.

I also note that https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations says "A _single_ parameterless annotation _may_ instead appear together with the first line of the signature" i.e. "_may_" not "_must_" and that the Lucene code with the same configuration has a mix of prior and same line `@Rule` style.

Reply:
Thanks for researching this, I think at the end of the day I don't care what the rule is as long as it is easy to enforce and validate.